requirements/deploy.in

```diff
@@ -0,0 +1,12 @@
+# Requirements for our production systems
+
+-r pip.txt
+
+psycopg2
+gunicorn
+django-redis-cache
+
+# For resizing images
+pillow
```

Comment:
Used to be pinned as `pillow==9.1.0`, this one might be less risky to pin loosely?

Reply:
(the update results in Pillow being bumped to 9.2.0)